### ISSUE TITLE
Add setting to ME stocking bus and hatch to automatically fetch the largest item stacks available in the network

### ISSUE THREAD

**Issue by Lukeb28** [GregTechCEu/GregTechCEu-1.19#2533]
### Cross-mod Integration

ME2

### Feature Description

Currently the ME stocking bus and hatch have a feature to automatically set their filters. This seems to be random however. If it could be configurable between random and largest stacks available, then utility with parallelization goes up as it can be part of a sub-net  for ore processing in which the most backed up ore is given priority. This would increase machine utilization by always prioritizing item stacks that can be done in parallel the best.

**Comment by omergunr100** [comment]
duplicate of #2342

**Comment by screret** [comment]
closing as duplicate
